src/main/java/com/spotinst/sdkjava/model/repo/azure/statefulNode/SpotinstAzureStatefulNodeRepo.java

```diff
@@ -0,0 +1,168 @@
+package com.spotinst.sdkjava.model.repo.azure.statefulNode;
+
+import com.spotinst.sdkjava.enums.AzureStatefulNodeStateEnum;
+import com.spotinst.sdkjava.exception.ExceptionHelper;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.ISpotAzureStatefulNodeRepo;
+import com.spotinst.sdkjava.model.RepoGenericResponse;
+import com.spotinst.sdkjava.model.api.azure.statefulNode.ApiStatefulNode;
+import com.spotinst.sdkjava.model.api.azure.statefulNode.ApiStatefulNodeDeallocationConfig;
+import com.spotinst.sdkjava.model.api.azure.statefulNode.ApiStatefulNodeGetStatusConfig;
+import com.spotinst.sdkjava.model.bl.azure.statefulNode.StatefulNode;
+import com.spotinst.sdkjava.model.bl.azure.statefulNode.StatefulNodeDeallocationConfig;
+import com.spotinst.sdkjava.model.bl.azure.statefulNode.StatefulNodeGetStatusConfig;
+import com.spotinst.sdkjava.model.converters.azure.statefulNode.StatefulNodeConverter;
+import com.spotinst.sdkjava.model.requests.azure.statefulNode.ApiStatefulNodeStateRequest;
+import com.spotinst.sdkjava.model.requests.azure.statefulNode.StatefulNodeStateRequest;
+import com.spotinst.sdkjava.model.service.azure.statefulNode.SpotinstAzureStatefulNodeService;
+import com.sun.org.apache.xpath.internal.operations.Bool;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class SpotinstAzureStatefulNodeRepo implements ISpotAzureStatefulNodeRepo {
+
+    @Override
+    public RepoGenericResponse<StatefulNode> createNode(StatefulNode NodeToCreate, String authToken, String account) {
```

Comment:
Why the input parameter and response type classes are same?

Reply:
CreateAPI Response are attributes we use for Create API.

Reply:
Both are same, so will reuse that same class for response also.